Assignment-2/app/src/Component/DisplayTaskList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Task1 from '../Component/taskComponents/Task1';
+import Task2 from '../Component/taskComponents/Task2';
+import Task3 from '../Component/taskComponents/Task3';
+import Task4 from '../Component/taskComponents/Task4';
+import Task5 from '../Component/taskComponents/Task5';
+import Task6 from '../Component/taskComponents/Task6';
+import Task7 from '../Component/taskComponents/Task7';
+import Task8 from '../Component/taskComponents/Task8';
+
+
+const DisplayTaskList = () => {
+  return (
+    <div>
+      <div style={{fontSize:'60px',display:'flex',justifyContent:'center'}}>TASK</div>
+      <ol>
+       <Task1/>
```

Comment:
I think we can use map here instead of creating tasks seperatly

Reply:
ok but need to store in the array all the separate task then we can render the task using map

Reply:
No, I don't think so. All the task component are same just they have different task string that can be passed as a prop.
Please connect with me for more clarity

Reply:
> I think we can use map here instead of creating tasks separately
also worked on this